SpatialGDK/Extras/internal-documentation/release-process.md

```diff
@@ -24,10 +30,12 @@ This document outlines the process for releasing a version of the GDK for Unreal
 1. Select "Continue" and move onto the next step.
 1. Wait for the "Prepare the release" step to run, it takes about 45 minutes, maybe grab a coffee?
 1. Once the "Prepare the release" step has passed, notify `#unreal-eng-safespace` that: "Release candidates have been cut, feel free to merge into master again"
-1. The "Build & upload all UnrealEngine release candidates" step will now commence.<br> While those builds run, take a look at the top of the build page, where you'll notice a new [annotation](https://buildkite.com/docs/agent/v3/cli-annotate): "your human labour is now required to complete the tasks listed in the PR descriptions and unblock the pipeline to resume the release."<br>Click through to the PRs using the links in the annotations and follow the steps. Come back when you're done.
-1. As soon as the "Build & upload all UnrealEngine release candidates" step has passed, select "Run all tests".
-1. Once all test have passed, all PRs are approved and all tasks listed in the PR descriptions are complete, select "Unblock the release". This will trigger "Release `ci/release.sh`".
-1. When "Release `ci/release.sh`" is complete, the unrealgdk-release pipeline will pass.<br>
+1. **IMPORTANT TEMP STEP** You must now update all the `rc` branches in all repositories. You must manually update the `yaml.steps` files to use the correct `rc` branches for a given release (e.g. `0.14.0-rc`). This is to get builds working. Additionally, you will need to update the default branches for building in `UnrealGDKBuild` `prepare_steps.py`. This is to ensure the created rc branches in new CI build the correct repo versions.
+3. The "Build & upload all UnrealEngine release candidates" step will now commence.<br> While those builds run, take a look at the top of the build page, where you'll notice a new [annotation](https://buildkite.com/docs/agent/v3/cli-annotate): "your human labour is now required to complete the tasks listed in the PR descriptions and unblock the pipeline to resume the release."<br>Click through to the PRs using the links in the annotations and follow the steps. Come back when you're done.
+4. As soon as the "Build & upload all UnrealEngine release candidates" step has passed, select "Run all tests".
+5. Once all test have passed, all PRs are approved and all tasks listed in the PR descriptions are complete, select "Unblock the release". This will trigger "Release `ci/release.sh`".
```

Comment:
Actually, I realised the documentation here has probably been wrong for a while.
First you do "Prepare full release", and only the last step is the actual release.
So here, 5. should refer to prepare full release.
Then 7. should be "trigger release and wait for it to complete" (but also Mike's suggestion, all of these should be 1.)

Reply:
But we already cover that above no?

Reply:
I meant this:
![image](https://user-images.githubusercontent.com/2778498/124958750-67c61500-e012-11eb-807c-3707e42fcfef.png)
First, we should do a "Prepare release candidate branches for full release", instead of what is above labelled as step 5, and then step 7 should be the actual "Unblock the release".